netbox/dcim/filtersets.py

```diff
@@ -1980,7 +1980,7 @@ class CableFilterSet(TenancyFilterSet, NetBoxModelFilterSet):
         method='_unterminated',
         label=_('Unterminated'),
     )
-    type = django_filters.MultipleChoiceFilter(
+    type = NullableMultipleChoiceFilter(
```

Comment:
This addresses the `type` field on Cable specifically, but I think what we need is a general implementation which covers _all_ non-required choice fields, across all models. Ideally, this should be handled automatically for all model fields with a `choices` attribute defined, which would obviate the need to manually declare these on the FilterSets as we currently do. (Granted, I'm not sure whether that's feasible.)